authors/apps/authentication/tests/test_update.py

```diff
@@ -12,7 +12,7 @@ class UpdateTest(BaseTest):
 
     def setUp(self):
         super().setUp()
-        self.url_update = reverse('authentication:retrieve_update')
+        self.url_update = reverse('authentication:retrieve-update')
```

Comment:
Why the change in url?

Reply:
this was done to follow the naming convention in the url namespaces. we are using the hypen(-) instead of the underscore(_)

Reply:
I realize this required a separate PT story and shall follow the correct procedure the next time but just like @missvicki explained, we have agreed to follow the same convention of using hyphens and not underscores in our url-names.